examples/howto/Model_Builder.myst.md

```diff
@@ -0,0 +1,255 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Using ModelBuilder class for deploying PyMC models 
+:::{post} Sep 12, 2022
+:tags: model builder class, model class, linear model
```

Comment:
We don't use classes (or any pymc objects for that matter) as tags. I would probably use something like "deployment" and other related topics.